git-branchless/src/commands/reword.rs

```diff
@@ -522,6 +527,14 @@ fn parse_bulk_edit_message(
         commits_oids.insert(commit.get_short_oid()?, commit.get_oid());
     }
 
+    let message = if commits.len() == 1 && !message.contains("++ reword") {
```

Comment:
To avoid the `.unwrap` here, we can write it something like this:

```
let message = match commits.as_slice() {
    [only_commit] if !message.contains("++ reword") => format!(...),
    _ => message,
};
```

Reply:
Oh ... that's yummy! Thank you! I've made this change. `match` is really exciting to learn more about. It's been eye opening. 👀